casepro/backend/junebug.py

```diff
@@ -109,9 +109,25 @@ def __init__(self, json_data):
         if remote_language is not None:
             self.language, _, _ = remote_language.partition('_')
         self.name = json_data.get('details').get('name', None)
-        self.addresses = json_data.get('details').get('addresses')
         self.fields = {}
         self.groups = {}
+        addresses = json_data.get('details').get('addresses')
+        self.urns = []
+        for scheme, address in addresses.items():
+            scheme_addresses = []
+            for urn, details in address.items():
+                if 'optedout' in details and details['optedout'] is True:
+                    # Skip opted out URNs
+                    continue
+                if 'default' in details and details['default'] is True:
+                    # If a default is set for the scheme then only store the default
+                    scheme_addresses = [urn]
+                    break
+                scheme_addresses.append(urn)
+            for value in scheme_addresses:
+                if scheme == "msisdn":
+                    scheme = "tel"
+                self.urns.append(scheme+":"+value)
```

Comment:
We either need spaces around the `+`, or we can use `'%s:%s' % (...)` or `.format`.